docs/Using-Fleet/ChromeOS.md

```diff
@@ -0,0 +1,23 @@
+# ChromeOS
+
+## Adding ChromeOS hosts to Fleet
+Fleet provides a Chrome extension which you can install via Google Admin.
+
+To learn how to add ChromeOS hosts to Fleet, visit [here](https://fleetdm.com/docs/using-fleet/adding-hosts#add-chromebooks-with-the-fleetd-chrome-extension).
+
+## Available tables
+To see the available tables for ChromeOS, visit [here](https://fleetdm.com/tables).
+
+## Setting the hostname
+By default, the hostname for a Chromebook host will be blank. The hostname can be customized in Google Admin by configuring the ["device network hostname template"](https://support.google.com/chrome/a/answer/1375678#zippy=%2Cdevice-network-hostname-template%2Creport-device-os-information).
```

Comment:
If we want to specify the drill down, we can add to that second sentence. I had to use `cmd + f` to find "device network hostname template" in both the settings and the docs since it wasn't clear.

"The hostname can be customized in Google Admin by  Under Devices > Chrome > Settings > Device > Device Settings > Other Settings, configure ["Device network hostname template"](https://support.google.com/chrome/a/answer/1375678#zippy=%2Cdevice-network-hostname-template%2Creport-device-os-information)."

Screenshot of drill down:
<img width="1477" alt="Screenshot 2023-06-09 at 9 34 24 AM" src="https://github.com/fleetdm/fleet/assets/71795832/a955fa38-6845-4058-8da1-9feb92b09525">

Reply:
```suggestion
By default, the hostname for a Chromebook host will be blank. The hostname can be customized in Google Admin under Devices > Chrome > Settings > Device > Device Settings > Other Settings > [Device network hostname template](https://support.google.com/chrome/a/answer/1375678#zippy=%2Cdevice-network-hostname-template%2Creport-device-os-information).
```